easybuild/easyblocks/c/cgal.py

```diff
@@ -42,26 +45,31 @@ class EB_CGAL(CMakeMake):
     def configure_step(self):
         """Set some extra environment variables before configuring."""
 
-        deps = ["Boost", "GMP", "MPFR"]
+        deps = ['Boost', 'GMP', 'MPFR']
         for dep in deps:
             if not get_software_root(dep):
-                raise EasyBuildError("Dependency module %s not loaded?", dep)
+                raise EasyBuildError('Dependency module %s not loaded?', dep)
 
-        for lib in ["GMP", "MPFR"]:
-            os.environ['%s_INC_DIR' % lib] = "%s%s" % (get_software_root(lib), "/include/")
-            os.environ['%s_LIB_DIR' % lib] = "%s%s" % (get_software_root(lib), "/lib/")
+        for lib in ['GMP', 'MPFR']:
+            os.environ['%s_INC_DIR' % lib] = os.path.join(get_software_root(lib), 'include')
+            os.environ['%s_LIB_DIR' % lib] = os.path.join(get_software_root(lib), 'lib')
 
-        os.environ['BOOST_ROOT'] = get_software_root("Boost")
+        os.environ['BOOST_ROOT'] = get_software_root('Boost')
 
         super(EB_CGAL, self).configure_step()
 
     def sanity_check_step(self):
         """Custom sanity check for CGAL."""
+
         shlib_ext = get_shared_lib_ext()
-        libdirs = ('lib', 'lib64')
-        libs = [tuple(os.path.join(d, 'libCGAL%s.%s' % (l, shlib_ext)) for d in libdirs) for l in ['', '_Core']]
+        libs = [os.path.join('lib64', 'libCGAL%s.%s' % (l, shlib_ext)) for l in ['', '_Core']]
+        dirs = [os.path.join('include', 'CGAL')]
+        if LooseVersion(self.version) >= LooseVersion('4.12'):
+            dirs += os.path.join('lib64', 'cmake', 'CGAL')
+        else:
+            dirs += os.path.join('lib64', 'CGAL')
```

Comment:
@damianam please use `dirs.append(...)` here to add an item to the list, same above